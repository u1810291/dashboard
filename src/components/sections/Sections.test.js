import React from 'react'
import { shallow } from 'enzyme'
import Section from '.'

describe('Sections', () => {
  test('component render', () => {
    const component = shallow(
      <Section>
        <div />
        <p />
      </Section>
    )
    expect(component).toMatchSnapshot()
  })
})
