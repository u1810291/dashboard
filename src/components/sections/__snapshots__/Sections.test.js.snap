// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sections component render 1`] = `
<div
  className=""
>
  <div />
  <p />
</div>
`;
